### ISSUE TITLE
Problem with GetSoundWaveFromFile

### ISSUE THREAD

**Issue by jdounivers** [EverNewJoy/VictoryPlugin#67]
Hello,

I need to play sound from files (dynamic data for my games), but each times i try to use this method (GetSoundWaveFromFile), the Return Value is _None_.
- I debbuged the filepath to be sure i give a correct param.
- I tried to put a literal string into
- I tried to make it relative to the project directory, then to the content directory.
But nothing works.
Is there something i missed on this function ?

Thank you in advance for your help, i really need it :(
And thank you for your plugin it help a lot.

**Comment by CANA-Dan** [comment]
yeah i noticed this issue as well. an alternitive atm is audio analysis tools. it allows you to load in a multitude of diffrent file types at runtime https://github.com/gtreshchev/AudioAnalysisTools
